vocab.py

```diff
@@ -25,3 +25,4 @@ class ValidDataSourceName(Enum):
     CANVAS_API = auto()
     UNIZIN_DATA_PLATFORM_EVENTS = auto()
     UNIZIN_DATA_WAREHOUSE = auto()
+    CANVAS_ZOOM_MEETINGS = auto()
```

Comment:
I think it's a little confusing that this is the same as the `ValidJobName`. The data is coming from the Canvas API and the user interface, correct? I know that might change, but I'd probably create two `data_source_status` dictionaries, one for `CANVAS_API` and `CANVAS_UI`. What do you think?

Reply:
Yeah, the source of the data is both CANVAS_API and from the ZOOM_UI (or maybe ZOOM_LTI). I feel like it's more just ZOOM in this case really. Maybe it could be both of these though. I don't 100% understand how this is represented in the data model as I haven't looked at that yet.

Reply:
I'll leave it up to you, but the data source(s) should document everywhere data is coming from. I get that it's more confusing with LTI, so whatever works for you.